docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/utils/Functions.java

```diff
@@ -0,0 +1,46 @@
+package com.nirima.jenkins.plugins.docker.utils;
+
+import com.nirima.jenkins.plugins.docker.DockerTemplateBase;
+import com.nirima.jenkins.plugins.docker.launcher.DockerComputerJNLPLauncher;
+import com.nirima.jenkins.plugins.docker.launcher.DockerComputerSSHLauncher;
+import com.nirima.jenkins.plugins.docker.strategy.DockerCloudRetentionStrategy;
+import com.nirima.jenkins.plugins.docker.strategy.DockerOnceRetentionStrategy;
+import hudson.model.Descriptor;
+import hudson.slaves.ComputerLauncher;
+import hudson.slaves.RetentionStrategy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * UI helper class.
+ *
+ * @author Kanstantsin Shautsou
+ */
+public class Functions {
```

Comment:
So if u wanna use `hudson.Functions`, you can't use import - you should write full class name